### PR TITLE
Rupert/additional function inputs

### PR DESCRIPTION
**Pull request by ru4ert**
Many thanks for submitting your Pull Request :heart:!

Please make sure that your PR meets the following requirements:

- [x] You have read the [contributors guide](CONTRIBUTING.md)
- [x] Your code is properly formatted according to [our code style](CONTRIBUTING.md#code-style)
- [x] Pull Request title contains the target branch if not targeting main: `[0.9.x] Subject`
- [x] Pull Request contains link to the discussion
- [x] Pull Request contains link to any dependent or related Pull Request
- [x] Pull Request contains description of the issue
- [x] Pull Request does not include fixes for issues other than the main ticket

https://github.com/quarkiverse/quarkus-openapi-generator/discussions/625#discussioncomment-8133645

### CODE REVIEW

**Comment by mcruzdev** [comment]
Hi @ru4ert, thank you! Looks pretty good!

Can you add documentation to the new configuration?

File: deployment/src/main/java/io/quarkiverse/openapi/generator/deployment/CommonItemConfig.java [hunk 1]
```diff
@@ -48,6 +48,12 @@ public class CommonItemConfig {
     @ConfigItem(name = "additional-api-type-annotations")
     public Optional<String> additionalApiTypeAnnotations;
 
+    /**
+     * Add custom/additional HTTP Headers or other args to every request
+     */
+    @ConfigItem(name = "additional-request-args")
+    public Optional<String> additionalRequestArgs;
```
Comment (mcruzdev) [inline deployment/src/main/java/io/quarkiverse/openapi/generator/deployment/CommonItemConfig.java:55]: IHO, I think that the property could be `additional-request-headers` or something like it. WYT @hbelmiro?

Comment (ru4ert) [reply deployment/src/main/java/io/quarkiverse/openapi/generator/deployment/CommonItemConfig.java:55]: @ru4ert I think your not limited to just headers. See my Test

**Comment by mcruzdev** [comment]
Hi @ru4ert thank you? I think that you need to use the page `docs/modules/ROOT/pages/includes/additional-request-args.adoc` into the `index.adoc`.

File: deployment/src/main/java/io/quarkiverse/openapi/generator/deployment/CommonItemConfig.java (hunk 1 above)
Comment (mcruzdev) [reply deployment/src/main/java/io/quarkiverse/openapi/generator/deployment/CommonItemConfig.java:55]: Makes sense!

**Review by mcruzdev — COMMENTED** [review]
Hi @ru4ert thank you!
I think that you need to use the page `docs/modules/ROOT/pages/includes/additional-request-args.adoc` into the `index.adoc`.

**Comment by mcruzdev** [comment]
LGTM! WYT @ricardozanini @hbelmiro? 

Good job @ru4ert!

**Review by mcruzdev — APPROVED** [review]

**Comment by mcruzdev** [comment]
Fixes #643

**Review by hbelmiro — APPROVED** [review]

**Comment by hbelmiro** [comment]
@all-contributors add @ru4ert for code and tests.

**Pull request merged**